quirc/README.md

```diff
@@ -0,0 +1,5 @@
+# Quirc QR code decoding library
+
+This is an ESP-IDF component for [quirc](https://github.com/dlbeer/quirc), a QR code decoding library.
+
```

Comment:
Would it make sense to cross-reference `qrcode` component for QR code generation?